lib/danger/danger_core/plugins/dangerfile_import_plugin.rb

```diff
@@ -36,41 +36,53 @@ def self.instance_name
     #           a danger plugin from.
     # @return [void]
 
-    def import(path)
-      raise "`import` requires a string" unless path.kind_of?(String)
-      path += ".rb" unless path.end_with?(".rb")
+    def import(path_or_url)
+      raise "`import` requires a string" unless path_or_url.kind_of?(String)
```

Comment:
this should be `is_a` instead of `kind_of` in case someone is using a String subclass